test/dynamo/test_dynamo.py

```diff
@@ -369,10 +377,14 @@ def get_loader(self, device, sample_count, batch_size=4):
   @skipOnTpu
   @skipOnNeuron
   @parameterized.parameters(
-      True,
-      False,
+      (True, 'openxla'),
+      (False, 'openxla'),
+      (True, dynamo_backend2.dynamo_backend),
+      (False, dynamo_backend2.dynamo_backend),
   )
-  def test_resnet18(self, initialize_on_cuda):
+  def test_resnet18(self, initialize_on_cuda, backend):
+    import torchax
+    torchax.default_env().config.debug_print_each_op = True
```

Comment:
Is this left-over debugging code or does it serve a purpose?

Reply:
yes, removed.